### PR TITLE
Init

### PR DESCRIPTION
**Pull request by bouboumael**

### CODE REVIEW

**Review by wonecode — DISMISSED** [review]

**Review by RedPandore — DISMISSED** [review]

**Review by RaphaelBS-WCS — DISMISSED** [review]
Approved!

**Review by jaldabaoth-code — DISMISSED** [review]

File: templates/base.html.twig [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
-		{% block body %}{% endblock %}
-	</body>
+        {% block javascripts %}
+            {{ encore_entry_script_tags('app') }}
+        {% endblock %}
+    </head>
+    <body>
+		{% block navbar %}{% endblock %}
+        {% block body %}{% endblock %}
+		{% block footer %}{% endblock %}
```
Comment (LucyMerlier) [inline templates/base.html.twig]: indentation?

File: templates/home/index.html.twig [hunk 2]
```diff
@@ -1,8 +1,7 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}Home{% endblock %}
+{% block title %}{{ parent() }} - Home {% endblock %}
```
Comment (LucyMerlier) [inline templates/home/index.html.twig]: "Home"? Is this an english speaking website?

File: templates/home/index.html.twig [hunk 3]
```diff
@@ -1,8 +1,7 @@
 {% extends 'base.html.twig' %}
```
Comment (LucyMerlier) [inline templates/home/index.html.twig]: so you have a "layout.html.twig", but this template extends "base.html.twig" directly?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: templates/home/index.html.twig (hunk 3 above)
Comment (bouboumael) [reply templates/home/index.html.twig]: done

**Review by jaldabaoth-code — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**